dhcpv6/types.go

```diff
@@ -224,12 +254,16 @@ const (
 	OptionRelayPort                               OptionCode = 135
 	OptionV6SZTPRedirect                          OptionCode = 136
 	OptionS46BindIPv6Prefix                       OptionCode = 137
-	_                                             OptionCode = 138
-	_                                             OptionCode = 139
-	_                                             OptionCode = 140
-	_                                             OptionCode = 141
-	_                                             OptionCode = 142
+	OptionIaLL                                    OptionCode = 138
```

Comment:
Just out of curiosity, are you planning to use any of these new v6 types?

Reply:
encrypted DNS option